front/app/containers/Admin/pagesAndMenu/containers/CustomPages/CustomPageSettingsForm.tsx

```diff
@@ -0,0 +1,99 @@
+import React, { useState } from 'react';
+import { Multiloc } from 'typings';
+
+// form
+import Feedback from 'components/HookForm/Feedback';
+import { FormProvider, useForm } from 'react-hook-form';
+import { SectionField } from 'components/admin/Section';
+import { yupResolver } from '@hookform/resolvers/yup';
+import { object, string } from 'yup';
+import validateMultiloc from 'utils/yup/validateMultiloc';
+import { slugRegEx } from 'utils/textUtils';
+// import { handleHookFormSubmissionError } from 'utils/errorUtils';
+
+// components
+import SectionFormWrapper from 'containers/Admin/pagesAndMenu/components/SectionFormWrapper';
+import Button from 'components/UI/Button';
+import Input from 'components/HookForm/Input';
+import InputMultilocWithLocaleSwitcher from 'components/HookForm/InputMultilocWithLocaleSwitcher';
+import { Box } from '@citizenlab/cl2-component-library';
+
+// intl
+import messages from './messages';
+import { InjectedIntlProps } from 'react-intl';
+import { injectIntl } from 'utils/cl-intl';
+
+// services
+import { createCustomPageStream } from 'services/customPages';
+
+interface CreateCustomPageFormValues {
+  title_multiloc: Multiloc;
+  slug: string;
+}
+
+interface Props {}
+
+const CustomPageSettingsForm = ({
+  intl: { formatMessage },
+}: Props & InjectedIntlProps) => {
+  // types still to change
+  const [_error, setError] = useState({});
+  const schema = object({
+    title_multiloc: validateMultiloc(
+      formatMessage(messages.titleMultilocError)
+    ),
+    slug: string()
+      .matches(slugRegEx, formatMessage(messages.slugRegexError))
+      .required(formatMessage(messages.slugRequiredError)),
+  });
+
+  const methods = useForm({
+    mode: 'onBlur',
+    resolver: yupResolver(schema),
+  });
+
+  const onFormSubmit = async (formValues: CreateCustomPageFormValues) => {
+    try {
+      createCustomPageStream(formValues);
+    } catch (error) {
+      setError(error);
+    }
+  };
+
```

Comment:
Not complete yet, hooking up the API will be a different ticket. This is just about the UI.